src/index.html

```diff
@@ -22,6 +22,431 @@
+            src="./images/logo.png"
+            alt="Moyo logo"
+            class="logo__img"
+          />
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link is-active"
+              >
+                apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                smartphones
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                laptops & computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                gadgets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                tablets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                photo
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="cards-container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image">
+          <a
+            href="#"
+            class="image"
+          ></a>
+        </div>
+
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__product-code">Product code: 195434</div>
+
+        <div class="card__reviews">
+          <div class="stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <div class="reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="price">Price:</div>
+          <div class="amount">$2,199</div>
+        </div>
+
+        <div class="button">
+          <a
+            class="button__buy"
+            href="#"
```

Comment:
The `href` attribute is set to '#', which is a placeholder. Consider providing a meaningful URL or using JavaScript to handle the click event if this is meant to be a dynamic link.